assets/sass/v2/_device-challenge.scss

```diff
@@ -0,0 +1,27 @@
+.siw-main-view {
+  .device-challenge-poll {
+    .o-form-content {
+      .okta-form-subtitle {
+        font-weight: 600;
+      }
+    }
+    .o-form-fieldset-container {
+      width: 88%;
```

Comment:
curious why would need to style the container?

Reply:
to make the text container narrower and text to show in more lines

Reply:
To make the text container narrower so that text is displayed in more lines.